app/stuquiz/controllers/university/get_universities_controller.py

```diff
@@ -0,0 +1,26 @@
+# @author Simone Nicol <en0mia.dev@gmail.com>
+# @created 04/07/23
+import json
+from typing import Optional
+
+from flask import Response
+
+from app.stuquiz.controllers.abstract_controller import AbstractController
+from app.stuquiz.models.university.university_model import UniversityModel
+
+
+class GetUniversitiesController(AbstractController):
+    def __init__(self, university_model: Optional[UniversityModel] = None):
+        self.university_model = university_model or UniversityModel()
+
+    def execute(self, data: dict) -> Response:
+        """
+        :param data: an empty dict
+        :return: HTTP Response
+        """
+        universities = self.university_model.get_universities()
+        result = []
+
+        for university in universities:
```

Comment:
In this case you can compact the code using list comprehension.

`result = json.dumps([university.dump() for university in universities])
return Response(result, 200)`

Reply:
Done, thanks!